packages/components/stories/toggle-switch/README.md

```diff
@@ -0,0 +1,48 @@
+# Toggle Switch Component
+
+Toggle switche is a digital on/off switch. It prompt users to choose between two mutually exclusive options and always have a default value. 
```

Comment:
Please keep descriptions simple and short.
Here we can say: 
A switch component with default value.

```suggestion
Toggle switch is a digital on/off switch. It prompt users to choose between two mutually exclusive options and always have a default value. 
```